orchestrator/src/test/java/com/scottlogic/deg/orchestrator/endtoend/JarExecuteTests.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.orchestrator.endtoend;
+
+import org.junit.jupiter.api.Test;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JarExecuteTests {
+
+    @Test
+    void GenerateSuccessfullyFromJar() throws Exception {
+            ProcessBuilder pb = new ProcessBuilder("java", "-jar", "build/libs/generator.jar", "generate","-p=src/test/java/com/scottlogic/deg/orchestrator/endtoend/testprofile.profile.json","--max-rows=1");
+            Process p = pb.start();
+            BufferedReader BufferedSTDOUTReader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            List<String> STDOUT = new ArrayList<String>();
+            String line;
+            while ((line = BufferedSTDOUTReader.readLine()) != null) {
+                STDOUT.add(line);
+            }
+            p.waitFor();
+            p.destroy();
+
+            assertEquals(2,STDOUT.size());
+            assertEquals("\"Generation successful\"",STDOUT.get(1));
```

Comment:
Do you think we should test that the first line is the expected field header?

Reply:
Potentially - I also had some testing of the STDERR but took it out as was unsure if I had too many asserts. One issue with the current setup is that if generation fails the test fails but does not provide any useful information about why as that's printed in the STDERR not the STDOUT

Reply:
That's true, but if it fails on CircleCI you can always try to run it locally, in which case you should see STDERR too.

One approach might be to capture both stdout and stderr, construct a command line that should (on success) not output anything to stderr, and then assert that nothing is output to stderr.  If anything is, you could output it as part of your assertion failure

Reply:
I have adjusted the test to assert both the correct field header and data are output. 

What command line options coulf be used that would prevent production of STDERR messages on success (currently Gen start time: , velocity table, Gen end time)

Reply:
The `--quiet` option should do

Reply:
Done - STDERR is now printed on STDOUT (only if test fails) so error message can be seen in failed test description